paasta_tools/deployd/master.py

```diff
@@ -29,6 +29,33 @@
 from paasta_tools.utils import ZookeeperPool
 
 
+class BounceQueue(PaastaQueue):
```

Comment:
What do you think about calling this "DedupedQueue"?

Also, is there any reason why this shouldn't be part of the main queue class and used for every queue? In what situation in this deployd use-case where we want duplicates anyway?